yt/visualization/color_maps.py

```diff
@@ -35,6 +35,12 @@ def check_color(name):
 yt_colormaps = {}
 
 def add_cmap(name, cdict):
+    """Deprecated alias, kept for backwards compatibility."""
+    from yt.funcs import issue_deprecation_warning
```

Comment:
I think this import is small enough it can move up to the top of the file with the other import statements.